### PR TITLE
docs(ava): remove redundant comments and assign 3.0.0 todo to developers

### PR DESCRIPTION
**Pull request by pddpd**
ATT

### CODE REVIEW

File: packages/ava/src/advisor/advise-pipeline/data-to-advices.ts [hunk 1]
```diff
@@ -22,8 +22,8 @@ import type {
 
 /**
  * options for advising inner pipeline
- * @todo refactor since 3.0.0
  */
+// TODO @neoddish: refactor since 3.0.0
```
Comment (pddpd) [inline packages/ava/src/advisor/advise-pipeline/data-to-advices.ts:26]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/advisor/advise-pipeline/data-to-advices.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ function scoreRules(
 
   // Hard-Rule pruning
   // holding for showLog @deprecated and testing
-  /** @since 3.0.0 @todo */
+  // @since 3.0.0 TODO @neoddish
```
Comment (pddpd) [inline packages/ava/src/advisor/advise-pipeline/data-to-advices.ts:59]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/advisor/advise-pipeline/data-to-advices.ts [hunk 3]
```diff
@@ -65,7 +65,7 @@ function scoreRules(
 
   const softScore = computeScore(chartType, chartWIKI, ruleBase, 'SOFT', info, log);
 
-  /** @since 3.0.0 @todo score normalization  */
+  // @since 3.0.0 TODO @neoddish score normalization
```
Comment (pddpd) [inline packages/ava/src/advisor/advise-pipeline/data-to-advices.ts:68]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/advisor/ruler/rules/limit-series.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ function hasSeriesField(dataProps: BasicDataPropertyForAdvice[]): boolean {
   return nominalOrOrdinalFields.length >= 2;
 }
 
-// TODO: 重构规则逻辑
+// TODO @neoddish: 重构规则逻辑
```
Comment (pddpd) [inline packages/ava/src/advisor/ruler/rules/limit-series.ts:13]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/advisor/ruler/rules/limit-series.ts [hunk 5]
```diff
@@ -40,14 +40,15 @@ export const limitSeries: RuleModule = {
...
+        // TODO @Guangmingyoubei: 确认下这个注释还需要吗
         // const f1 = sortedFields[0];
         const f2 = sortedFields[1];
 
         if (f2.distinct) {
           result = 1 / f2.distinct;
 
           if (f2.distinct > 6 && chartType === 'heatmap') {
-            // TODO 为什么 result 可以是 2？
+            // TODO @neoddish: 为什么 result 可以是 2？
```
Comment (pddpd) [inline packages/ava/src/advisor/ruler/rules/limit-series.ts:51]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/ckb/base.ts [hunk 6]
```diff
@@ -1353,8 +1353,8 @@ export const base: PureChartKnowledgeBase = {
 
   /**
    * WIP: unavailable until completed
-   * @todo add channel 'Text'
    */
+  // TODO @neoddish: add channel 'Text'
```
Comment (pddpd) [inline packages/ava/src/ckb/base.ts:1357]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/ckb/base.ts [hunk 7]
```diff
@@ -1372,8 +1372,8 @@ export const base: PureChartKnowledgeBase = {
...
```
Comment (pddpd) [inline packages/ava/src/ckb/base.ts:1376]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/ckb/index.ts [hunk 8]
```diff
@@ -1,4 +1,4 @@
-// TODO: todo list for ckb
+// TODO @neoddish: todo list for ckb
```
Comment (pddpd) [inline packages/ava/src/ckb/index.ts:1]: would you finish it before the release of v3? @neoddish

File: packages/ava/src/advisor/ruler/rules/limit-series.ts [hunk 9]
```diff
@@ -40,14 +40,15 @@ export const limitSeries: RuleModule = {
       if (nominalOrOrdinalFields.length >= 2) {
         const sortedFields = nominalOrOrdinalFields.sort(compare);
 
+        // TODO @Guangmingyoubei: 确认下这个注释还需要吗
```
Comment (pddpd) [inline packages/ava/src/advisor/ruler/rules/limit-series.ts]: would you finish it before the release of v3? @GuangMingYouBei

File: packages/ava/src/advisor/utils/index.ts [hunk 10]
```diff
@@ -1,5 +1,5 @@
-/**
- * TODO： migrate utils into one unified folder of in ava.
+/*
+ * TODO @GuangMingYouBei: migrate utils into one unified folder of in ava.
```
Comment (pddpd) [inline packages/ava/src/advisor/utils/index.ts:2]: would you finish it before the release of v3? @GuangMingYouBei

File: packages/ava/src/insight/pipeline/extract.ts [hunk 11]
```diff
@@ -208,7 +208,7 @@ export function extractInsightsFromSubspace(
 
   /** insight extraction */
   const insights: (InsightInfo<PatternInfo> | null)[] = [];
-  // TODO Combination 1:  1D(dimension) or 1M(measure)
+  // TODO @Esorakouki: Combination 1:  1D(dimension) or 1M(measure)
```
Comment (pddpd) [inline packages/ava/src/insight/pipeline/extract.ts:211]: would you finish it before the release of v3? @Esorakouki

File: packages/ava/src/insight/visualization/index.ts [hunk 12]
```diff
@@ -38,7 +38,7 @@ export function getInsightVisualizationSchema(
   Object.entries(patternGroups).forEach(([chartType, patternGroup]: [string, PatternInfo[]]) => {
     const narrative = new InsightNarrativeGenerator(patterns, insight);
 
-    // TODO chart schema generation
+    // TODO @Esorakouki: chart schema generation
```
Comment (pddpd) [inline packages/ava/src/insight/visualization/index.ts:41]: would you finish it before the release of v3? @Esorakouki

File: packages/ava/src/insight/visualization/index.ts [hunk 13]
```diff
@@ -108,7 +108,7 @@ export function getHomogeneousInsightVisualizationSchema(
 
   patterns.forEach((pattern) => {
     const { insightType } = pattern;
-    // TODO chart schema generation
+    // TODO @Esorakouki: chart schema generation
```
Comment (pddpd) [inline packages/ava/src/insight/visualization/index.ts:111]: would you finish it before the release of v3? @Esorakouki

File: packages/ava/src/advisor/utils/index.ts (hunk 10 above)
Comment (GuangMingYouBei) [reply packages/ava/src/advisor/utils/index.ts:2]: nope

File: packages/ava/src/advisor/ruler/rules/limit-series.ts (hunk 9 above)
Comment (GuangMingYouBei) [reply packages/ava/src/advisor/ruler/rules/limit-series.ts]: nope

**Comment by neoddish** [comment]
all my todos: between alpha and 3.0.0

**Review by neoddish — APPROVED** [review]
LGTM

**Pull request merged**